src/main/java/graphql/annotations/connection/SimplePaginatedDataImpl.java

```diff
@@ -0,0 +1,16 @@
+package graphql.annotations.connection;
+
+public class SimplePaginatedDataImpl<T> extends AbstractSimplePaginatedData<T> {
+
+    private long overAll;
```

Comment:
whats overall means?

Reply:
the amount of all the entities that can be retrieved

Reply:
call it totalCount or overAllCount

Reply:
done